VaticanBAV/et/BAVet117.xml

```diff
@@ -26,29 +26,152 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+                                <dim unit="leaf">10</dim>
+                                <locus from="53r" to="62v"></locus>
+                              </item>
+                              <item xml:id="q8" n="7">
+                                <dim unit="leaf">10</dim>
+                                <locus from="63r" to="72v"></locus>
+                              </item>
+                              <item xml:id="q9" n="8">
+                                <dim unit="leaf">10</dim>
+                                <locus from="73r" to="82v"></locus>
+                              </item>
+                              <item xml:id="q10" n="9">
+                                <dim unit="leaf">10</dim>
+                                <locus from="83r" to="92v"></locus>
+                              </item>
+                            </list>
+                          </collation>
+                        </supportDesc>
+                        <layoutDesc>
+                          <layout columns="1" writtenLines="19"/>
+                        </layoutDesc>
+                      </objectDesc>
+                      <handDesc>
+                        <handNote xml:id="h1" script="Ethiopic">
+                          <seg type="rubrication">
+                            Titles
+                          </seg>
+                        </handNote>
+                      </handDesc>
+                      <additions>
+                        <list>
+                          <item xml:id="a1">
+                            <locus from="1r" to="4r"/>
+                            <desc>The scribe began to copy <title type="complete" ref="LIT1295Dersan"/>, leaving space to paint an accompanying image of the archangel Michael.</desc>
+                          </item>
+                          <item xml:id="a2">
+                            <locus from="1v" to="4v"/>
+                            <desc>Pen trials.</desc>
+                          </item>
+                          <item xml:id="a3">
+                            <locus target="#63r"/>
+                            <desc>A ፫ in the interior margin, used to number the quire, indicates that the manuscript was formed from two manuscripts.</desc>
+                          </item>
+                        </list>
+                      </additions>
+                      <bindingDesc>
+                         <binding contemporary="true">
+                            <decoNote xml:id="b1">Two wooden boards.</decoNote>
+                            <decoNote xml:id="b2" type="bindingMaterial">
+                             <material key="wood"/>
+                            </decoNote>
+                         </binding>
+                      </bindingDesc>
+                    </physDesc>
+                    <history>
+                     <origin>
+                      <origDate>
+                        <date notBefore="1900" notAfter="2000"/>
```

Comment:
`<date notBefore="1916" notAfter="1974"/>`? we can be somewhat more specific than the 20th century